eth/v1alpha1/validator.proto

```diff
@@ -15,13 +15,120 @@ syntax = "proto3";
 
 package ethereum.eth.v1alpha1;
 
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+import "eth/v1alpha1/block.proto";
+import "eth/v1alpha1/attestation.proto";
+
 option csharp_namespace = "Ethereum.Eth.v1alpha1";
 option go_package = "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1;eth";
 option java_multiple_files = true;
 option java_outer_classname = "ValidatorProto";
 option java_package = "org.ethereum.eth.v1alpha1";
 option php_namespace = "Ethereum\\Eth\\v1alpha1";
 
+// Beacon node validator API
+//
+// The beacon node validator API enables a validator to connect
+// and perform its obligations on the Ethereum 2.0 phase 0 beacon chain.
+service BeaconNodeValidator {
+    // Retrieves validator duties for the requested validators.
+    //
+    // The server returns a list of duties which are the actions should be performed by validators for a given epoch.
+    // Validator duties should be polled every epoch, but due to chain reorg of >MIN_SEED_LOOKAHEAD could occur,
+    // the validator duties could chain. For complete safety, it is recommended to poll at every slot to ensure
+    // validator is fully aware of any sudden chain reorg.
+    rpc GetDuties(DutiesRequest) returns (DutiesResponse) {
+        option (google.api.http) = {
+            get: "/eth/v1alpha1/validator/duties"
+        };
+    }
+
+    // Retrieves the latest valid beacon block to be proposed on the beacon chain.
+    //
+    // The server returns a new beacon block, without proposer signature, that can be
+    // proposed on the beacon chain. The block should be filled with all the necessary
+    // data for proposer to sign.
+    rpc GetBlock(BlockRequest) returns (Block) {
+        option (google.api.http) = {
+            get: "/eth/v1alpha1/validator/block"
+        };
+    }
+
+    // Sends the newly signed beacon block to beacon node to be broadcasted to the rest of the network.
```

Comment:
I wonder if this comment should be included here about propagating through the network...each of these API methods should have a single responsibility, be clear and concise about what they do, and shouldn't raise questions. The name should give enough context about all it does.

Reply:
Perhaps the propagation can be done elsewhere by some other service, but that's outside the scope of what proposing a block means.